packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,92 @@
+:local {
+  .form-input {
+    background-color: var(--terra-form-input-background-color, #fff);
+    background-image: var(--terra-form-input-focus-animation-background, none);
+    background-size: var(--terra-form-input-focus-background-start, 0 0);
```

Comment:
This is lint. `background-size` should come after `background-repeat`.

Reply:
Fixed, good catch. ac2c322a